custom/icds_reports/management/commands/rebuild_ccs_record_ucr.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+import time
+
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id, _build_indicators
```

Comment:
Generally bad practice to import "private" methods that are prefaced with `_`.

This is a very short term management command, so it should be fine now though